pandas/tests/indexes/test_base.py

```diff
@@ -1976,8 +1977,19 @@ def test_dropna(self):
         with tm.assertRaisesRegexp(ValueError, msg):
             pd.Index([1, 2, 3]).dropna(how='xxx')
 
+    def test_get_combined_index(self):
+        result = _get_combined_index([])
+        tm.assert_index_equal(result, Index([]))
 
-def test_get_combined_index():
-    from pandas.core.index import _get_combined_index
-    result = _get_combined_index([])
-    tm.assert_index_equal(result, Index([]))
+    def test_repeat(self):
+        repeats = 2
+        idx = pd.Index([1, 2, 3])
+        expected = pd.Index([1, 1, 2, 2, 3, 3])
+
+        result = idx.repeat(repeats)
+        tm.assert_index_equal(result, expected)
+
+        with tm.assert_produces_warning(FutureWarning,
+                                        check_stacklevel=False):
```

Comment:
Perhaps, but look at how many <a href="https://github.com/pandas-dev/pandas/search?utf8=%E2%9C%93&q=check_stacklevel%3DFalse">times</a> we have had to pass in `check_stacklevel=False`.  I don't necessarily object to specifying a stack level, but if we're unable to consistently specify the correct stack level, why are we using it in our tests?

BTW, I'm not trying to avoid showing my error traceback.  I'm just in the middle of another PR.